layouts/projects/section.html

```diff
@@ -2,4 +2,15 @@
   <script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.2.0/list.js"></script>
   <script src="//cdnjs.cloudflare.com/ajax/libs/numeral.js/1.4.5/numeral.min.js"></script>
   <script src="/assets/js/projects.min.js"></script>
+  <script>
```

Comment:
If this is something you need to do more than once, can you create a partial for it?

Reply:
Sure. I add it here because projects re-defined the footer_suffix which causes mine cannot be included. 
But again, this is only for dropdown, if we decide to use dropdown, I will add it to footer.html; if we decide to use collapsible I will remove it.